dcrrates/rateserver/README.md

```diff
@@ -0,0 +1,49 @@
+# RateServer
+
+RateServer is a central server for distributing Decred exchange rate data to
+any number of clients. It is particularly useful for DCRData instances behind a
+load balancer, though certainly not restricted to that application.
+
+### Build and Run
+
+For a module-enabled version of Go, getting started should be as simple as
+
+```
+cd path/to/dcrdata/dcrrates/rateserver
+go build
+./rateserver
+```
+
+See [DCRData's notes on Go](https://github.com/decred/dcrdata#building-dcrdata-with-go-111)
+for additional configuration details.
+
+### Using RateServer with DCRData
+
+To point DCRData to RateServer, you must specify DCRData's `ratemaster` and
+`ratecert` configuration options. `exchange-monitor` must also be enabled. Set
+`ratemaster` to the same value as RateServer's `listen` parameter (defaults to
```

Comment:
This is the only part that's a little confusing.  I might set rateserver's `listen` to `0.0.0.0:7778`, but with dcrdata the `ratemaster` would have to be the public IP or fqdn.